### PR TITLE
Load the spreadsheet and retrive unprocessed Invitations

### PR DESCRIPTION
**Pull request by fpersico**

### CODE REVIEW

File: src/google.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    try {
+      const response = await sheets.spreadsheets.values.update(
+        {
+          auth: jwtClient,
+          spreadsheetId,
+          range: `U${rowIndex}`,
+          valueInputOption: "RAW",
+          requestBody: {
+            range: `${PROCESSED_COLUMN_LETTER}${rowIndex}`,
+            values: [["TRUE"]],
```
Comment (cloudify) [inline src/google.ts]: this value is showing up in multiple places, we should use a constant here

File: src/index.ts [hunk 2]
```diff
@@ -27,4 +29,81 @@ const logger = createLogger({
   ],
 });
 
-logger.info("Application started");
+// The core function that start the processing of the Invitations.
+async function startProcessing(
+  googleClient: ReturnType<typeof createGoogleClient>,
+) {
+  logger.info("Retriving unprocessed invitations");
```
Comment (cloudify) [inline src/index.ts]:
```suggestion
  logger.info("Retreiving unprocessed invitations");
```

File: src/index.ts [hunk 3]
```diff
@@ -27,4 +29,81 @@ const logger = createLogger({
...
+  // Crete the google client
+  const googleClient = createGoogleClient(
+    clientEmail,
+    maybeGooglePrivateKey.value,
+    spreadsheetId,
+  );
+
+  // TODO: Define a better way to handle the loop.
+  // Move the interval to an enviroment variable.
+  setInterval(() => startProcessing(googleClient).then(() => 0), 5000);
```
Comment (cloudify) [inline src/index.ts]: a `while(true)` with a sleep should be enough

**Review by cloudify — CHANGES_REQUESTED** [review]

File: src/google.ts (hunk 1 above)
Comment (fpersico) [reply src/google.ts]: Done

File: src/index.ts (hunk 2 above)
Comment (fpersico) [reply src/index.ts]: Done

File: src/index.ts (hunk 3 above)
Comment (fpersico) [reply src/index.ts]: Done

File: src/utils.ts [hunk 4]
```diff
@@ -1,6 +1,10 @@
 import { none, Option, some } from "fp-ts/lib/Option";
 import fs from "fs";
 
+export function sleep(ms: number) {
```
Comment (cloudify) [inline src/utils.ts]: https://github.com/teamdigitale/io-ts-commons/blob/master/src/promises.ts#L12

**Review by cloudify — CHANGES_REQUESTED** [review]

File: src/utils.ts (hunk 4 above)
Comment (fpersico) [reply src/utils.ts]: Changed

**Review by cloudify — APPROVED** [review]

**Pull request merged**